translate/cloud-client/translate_v3_translate_text_with_glossary.py

```diff
@@ -18,26 +18,25 @@
 
 
 def translate_text_with_glossary(
-    text="YOUR_TEXT_TO_TRANSLATE",
     project_id="YOUR_PROJECT_ID",
-    glossary_id="YOUR_GLOSSARY_ID"
+    glossary_id="YOUR_GLOSSARY_ID",
 ):
     """Translates a given text using a glossary."""
 
     client = translate_v3.TranslationServiceClient()
-
-    contents = [text]
+    text_to_translate = "account"
     parent = client.location_path(project_id, "us-central1")
 
     glossary = client.glossary_path(
         project_id, "us-central1", glossary_id  # The location of the glossary
     )
 
-    glossary_config = translate_v3.types.TranslateTextGlossaryConfig(glossary=glossary)
+    glossary_config = translate_v3.types.TranslateTextGlossaryConfig(
+        glossary=glossary)
 
     # Supported language codes: https://cloud.google.com/translate/docs/languages
     response = client.translate_text(
-        contents,
+        contents=[text_to_translate],
```

Comment:
move `"account"` to be directly inlined.

Reply:
Actually, if it has to be "account". Leave it as a variable as this doesn't explain what you are doing.

Reply:
okay